library/src/main/resources/vueComponents.js

```diff
@@ -2,11 +2,40 @@
 
 Vue.component('individual-chart', {
     props: ['data', 'col', 'chartid'],
-    template: '<div>' +
-    '<div class="row" style="margin-top: 30px">' +
-    '<h4 v-bind:style="{color: col}">{{ data.name }} ({{ data.duration }}ms)</h4>' +
-    '</div>' +
-    '<p v-bind:id="chartid" v-if="data.duration <= 0">The test took no measurable time to complete</p>' +
-    '<div class="row" v-bind:id="chartid" v-else></div>' +
-    '</div>'
+    data: function () {
+        return {
+            expanded: false
+        }
+    },
+    template: `
+    <div class="test-class-container">
+        <div class="row">
+            <h4 v-bind:style="{color: col}">{{ data.name }} ({{ data.duration }}ms)</h4>
+        </div>
+        <p v-if="data.duration <= 0" v-bind:id="chartid">The test took no measurable time to complete</p>
+        <div v-else class="row chartrow" v-bind:id="chartid"></div>
+        <button type="button" class="btn" v-on:click="expanded = !expanded">{{expanded ? "▼" : "►"}}</button>
```

Comment:
These symbols do not work on my machine, so only a questionmark is displayed. (Google Chrome on Windows 10)